src/Middleware/HttpOverrides/src/CertificateForwardingFeature.cs

```diff
@@ -10,10 +10,12 @@ namespace Microsoft.AspNetCore.HttpOverrides;
 
 internal sealed class CertificateForwardingFeature : ITlsConnectionFeature
 {
+    private static readonly Task<X509Certificate2?> _nullCertificate = Task.FromResult<X509Certificate2?>(null);
```

Comment:
This isn't needed (IMO).
* it's only use in the catch block, so in a exceptional case
* `Task.FromResult` uses a (internally) cached task (for default-values, which is `null` for reference types)

Reply:
Agree with this